### PR TITLE
20_1 treelist: add description for ctrl + up/down arrows; extend Space description

### PR DESCRIPTION
**Pull request by pavel-2020**
[Trello card](https://trello.com/c/Q7w6dWMq/1487-add-description-for-keys-to-treelist-keyboard-navigation)

### CODE REVIEW

File: concepts/05 UI Components/DataGrid/75 Keyboard Support.md [hunk 1]
```diff
@@ -72,7 +73,7 @@ A user can interact with the UI component using the following keys:
     </tr>
     <tr>
         <td>Tab / Shift + Tab</td>
-        <td>Moves the focus one cell forward/back.<br /> Focuses the first/last cell in the next/previous row if cells in the current row run out.</td>
+        <td>Moves the focus one element forward/back.<br /> Focuses the first/last element in the next/previous row if elements in the current row run out.</td>
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/DataGrid/75 Keyboard Support.md]: "in the current row run out" Explain

File: concepts/05 UI Components/DataGrid/75 Keyboard Support.md [hunk 2]
```diff
@@ -45,8 +46,8 @@ A user can interact with the UI component using the following keys:
         <td>Selects the focused row and clears the selection of previously selected rows.<sup>3</sup></td>        
     </tr>
     <tr>
-        <td>Ctrl + &larr; &rarr; &uarr; &darr;</td>
-        <td>Navigates between the headers, filter panel, filter row, data area, and pager.</td>
+        <td>Ctrl + &uarr; &darr;</td>
+        <td>Navigates between column header, filter row, data area, filter panel, and pager.</td>
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/DataGrid/75 Keyboard Support.md]:
```suggestion
        <td>Navigates between a column header, filter row, data area, filter panel, and pager.</td>
```

File: concepts/05 UI Components/TreeList/70 Keyboard Support.md [hunk 3]
```diff
@@ -66,7 +67,7 @@ A user can interact with the UI component using the following keys:
     </tr>
     <tr>
         <td>Tab / Shift + Tab</td>
-        <td>Moves the focus one cell forward/back.<br /> Focuses the first/last cell in the next/previous row if cells in the current row run out.</td>
+        <td>Moves the focus one element forward/back.<br /> Focuses the first/last element in the next/previous row if cells in the current row run out.</td>
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/TreeList/70 Keyboard Support.md]: Same here

File: concepts/05 UI Components/TreeList/70 Keyboard Support.md [hunk 4]
```diff
@@ -79,6 +80,10 @@ A user can interact with the UI component using the following keys:
         <td>Ctrl + &rarr; / Ctrl + &larr;</td>
         <td>Expands/collapses the focused row.</td>
     </tr>
+    <tr>
+        <td>Ctrl + &uarr; / Ctrl + &darr;</td>
+        <td>Navigates between column header, filter row, data area, filter panel, and pager.</td>
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/TreeList/70 Keyboard Support.md]:
```suggestion
        <td>Navigates between a column header, filter row, data area, filter panel, and pager.</td>
```

**Review by dirk-pieterse — APPROVED** [review]

File: concepts/05 UI Components/DataGrid/75 Keyboard Support.md (hunk 1 above)
Comment (pavel-2020) [reply concepts/05 UI Components/DataGrid/75 Keyboard Support.md]: image
The next focused element after the circled one is 'Olivia'.

File: concepts/05 UI Components/TreeList/70 Keyboard Support.md (hunk 3 above)
Comment (pavel-2020) [reply concepts/05 UI Components/TreeList/70 Keyboard Support.md]: The explanation is the same.

**Pull request closed without merging**